src/blueapi/startup/simmotor.py

```diff
@@ -27,8 +27,8 @@ def __init__(
         super().__init__(
             name=name,
             readback_func=readback_func,
-            value=value,
-            delay=delay,
+            value=value,  # type: ignore
+            delay=delay,  # type: ignore
```

Comment:
What's the correct type for these, can we make the SynAxisWithMotionEvents correct?

Reply:
as discussed above with @callumforrester we do things the easy way with the ophyd-sync devices, as they aren't long for this codebase